tests/unit/test_repo.py

```diff
@@ -1,13 +1,76 @@
-import os
+from dvc.repo import Repo
+from dvc.scm.git import GitTree
+from dvc.system import System
+from dvc.utils.compat import fspath
 
-from tests.basic_env import TestDvc
 
+def test_is_dvc_internal(tmp_dir, dvc):
+    assert dvc.is_dvc_internal(fspath(tmp_dir / ".dvc" / "file"))
+    assert not dvc.is_dvc_internal("file")
 
-class TestIsDvcInternal(TestDvc):
-    def test_return_false_on_non_dvc_file(self):
-        path = os.path.join("path", "to-non-.dvc", "file")
-        self.assertFalse(self.dvc.is_dvc_internal(path))
 
-    def test_return_true_on_dvc_internal_file(self):
-        path = os.path.join("path", "to", ".dvc", "file")
-        self.assertTrue(path)
+def test_destroy(tmp_dir, dvc):
```

Comment:
moved `.destroy`, `.collect`, `.status` to the unit test module. the reason behind it is that we are testing the behavior of a single unit, in this case, a single method.

Reply:
Still look like functional tests to me. We recreate the whole scenario and call a method, which is a mirror of cli command.

Reply:
got it, will bring them back

Reply:
@Suor , for future references, how do you make the distinction between functional and unit tests?